cmd/remove.go

```diff
@@ -45,5 +50,19 @@ func findRepo(repoName string) (entry, error) {
 		}
 	}
 
-	return entry{}, errors.New("Repository not found.")
+	return entry{}, errors.New("repository not found")
+}
+
+func removeRepo(repoName string) error {
+	repo, err := findRepo(repoName)
+	if err != nil {
+		return fmt.Errorf("repository `%s` not found", repoName)
+	}
+
+	_, err = client.Repositories.Delete(ctx, repo.Owner, repo.Repo)
+	checkError(err)
```

Comment:
I think it'd be better if this function returned the error and we checked the error in `Run`

```go
if err != nil {
    return err
}
```